### PR TITLE
Static component mocks

### PR DESCRIPTION
**Pull request by ivanwills**
This pull request implements the ability for use-cases to include a template(s) as well as the standard data driven and attribute driven examples.

An example template is created for ux-accordion to show a template in action, obviously most other components should get template examples eventually.

Also implemented is showing using cases in tabs as a way to simplify having many use cases for the documentation page.

### CODE REVIEW

**Comment by ivanwills** [comment]
Added showing use-cases in tabbed display (just an idea to simplify display) and now processing components with templates. Static template still not compiling unfortunately.

File: src/components/ux-accordion/use-cases/default-three-accordions-static.hbs [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+	</ux-accordianitem>
+	<ux-accordianitem>
+		<ux-anchor>Item 2</ux-anchor>
+		<ux-content><h2>Second accordian item</h2></ux-content>
+	</ux-accordianitem>
+	<ux-accordianitem>
+		<ux-anchor>Item 3</ux-anchor>
+		<ux-content><h2>Third accordian item</h2></ux-content>
+	</ux-accordianitem>
+</ux-accordian>
```
Comment (ScottMaclure) [inline src/components/ux-accordion/use-cases/default-three-accordions-static.hbs]: accordian vs accordion?

**Comment by ivanwills** [comment]
BTW the help wanted is for getting template mocks being processed correctly by the build time RactiveF object.

**Comment by ivanwills** [comment]
This is now a working example showing a templated use case for ux-accordion. (problem turned out to be a typo)

**Comment by ScottMaclure** [comment]
Merge conflicts to resolve

**Comment by ivanwills** [comment]
This is now basically working except that when the tests are run the click event isn't propagating (even to the internal ux-anchor component). Any suggestions on what may be missing?

**Comment by ivanwills** [comment]
Removed my ux-top-bar work as it is also exposing issues that I don't want to deal with right now. (will have a new PR for that later)

**Pull request merged**